### PR TITLE
doc: update doc to add ibmcloud specific properties

### PR DESCRIPTION
**Pull request by wyuany**
Add more details to document to run e2e test on ibmcloud

Fixes: https://github.com/confidential-containers/cloud-api-adaptor/issues/1646

### CODE REVIEW

File: ibmcloud/IMPORT_PODVM_TO_VPC.md [hunk 1]
```diff
@@ -10,22 +10,31 @@ To simpify this process a script has been created to aid this. `ibmcloud/image/i
...
+Usage: ./import.sh <docker-image/qcow2-file> <vpc-region> [--bucket <name> --region <cos-region> --instance <cos-instance> --endpoint <cos-endpoint> --api <cloud-endpoint> --os <operating-system>]
+```
 
 ### Arguments
 
 The first two arguments are positional:
 
 1. The name of the container image to extract the qcow2 file, or the file itself
-1. The VPC region to create the image in (the process can be repeated for multiple regions)
+1. The VPC region to create the image in (the process can be repeated for multiple regions)
```
Comment (liudalibj) [inline ibmcloud/IMPORT_PODVM_TO_VPC.md:37]: please remove the space at the end.

**Review by liudalibj — APPROVED** [review]
Only one blanksapce need be removed,
Others LGTM.

File: test/e2e/README.md [hunk 2]
```diff
@@ -161,3 +161,32 @@ TEST_PROVISION=no \
...
+IKS_SERVICE_URL="https://containers.cloud.ibm.com/global"
+PODVM_IMAGE_ID="<podvm-image-uploaded-previously>"
+INSTANCE_PROFILE_NAME="bz2-2x8"
+PODVM_IMAGE_ARCH="s390x"
+IMAGE_PULL_API_KEY="<can-be-same-as-apikey>"
+CAA_IMAGE_TAG="4ac087db10409a04e6d46d20ad08c39ad25b0778"
+EOF
+```
+
+- For `INSTANCE_PROFILE_NAME`, if it's not secure execution, the value is started with "bz". If it's secure execution, the value is started with 'bz2e'. More values can be found through ibmcloud command `ibmcloud is instance-profiles`.
```
Comment (liudalibj) [inline test/e2e/README.md:190]: the value is started with "bz" -> the value is started with "bz2"

File: test/e2e/README.md [hunk 3]
```diff
@@ -161,3 +161,32 @@ TEST_PROVISION=no \
...
+APIKEY="<your-ibmcloud-apikey>"
+
+IAM_SERVICE_URL="https://iam.cloud.ibm.com/identity/token"
+VPC_SERVICE_URL="https://jp-tok.iaas.cloud.ibm.com/v1"
+IKS_SERVICE_URL="https://containers.cloud.ibm.com/global"
+PODVM_IMAGE_ID="<podvm-image-uploaded-previously>"
+INSTANCE_PROFILE_NAME="bz2-2x8"
+PODVM_IMAGE_ARCH="s390x"
+IMAGE_PULL_API_KEY="<can-be-same-as-apikey>"
+CAA_IMAGE_TAG="4ac087db10409a04e6d46d20ad08c39ad25b0778"
```
Comment (liudalibj) [inline test/e2e/README.md:186]: "4ac087db10409a04e6d46d20ad08c39ad25b0778" -> "\<caa-image-tag\>"

**Review by snir911 — APPROVED** [review]

**Review by huoqifeng — APPROVED** [review]
LGTM

**Pull request merged**